packages/integrations/vercel/README.md

```diff
@@ -172,6 +172,27 @@ import astroLogo from '../assets/logo.png';
 />
 ```
 
+### useSquooshDev
+
+**Type:** `boolean`<br>
+**Available for:** Serverless, Static
+**Added in:** `@astrojs/vercel@3.3.0`
+
+When enabled, in development mode a Squoosh-based image service will be used in development instead of Sharp. This is useful if you cannot install Sharp's dependencies on your development machine.
```

Comment:
```suggestion
Allows you to configure a Squoosh-based image service to be used in development instead of Sharp. This is useful if you cannot install Sharp's dependencies on your development machine.
```